lib/cocoapods/resolver.rb

```diff
@@ -2,6 +2,13 @@
 require 'cocoapods/resolver/lazy_specification'
 
 module Pod
+
```

Comment:
rm this line to make Rubocop :smile: (can also run `bundle exec rubocop --auto-correct lib`)